project/webapp/templates/partials/account/tabs/my_bills.html

```diff
@@ -1,26 +1,88 @@
-            <div class="section no-state">
-                <div class="container">
-                    <div class="section">
-                        <div class="center title-lato text">NO BILLS</div>
+    <div class="profile-my-bills-list">
+        <div class="row tab-title">
+            <div class="col s6 title-lato">My Bills</div>
+        </div>
+
+        <div class="section no-state">
+            <div class="container">
+                <div class="section">
+                    <!-- Populate Bills list -->
+                    <div class="subtitle-lato">Bills I support</div>
+                    <div class="row">
+                        <ul class="list-group">
+                            {{# _.each(this.model.get('my_bills').supported_bills, function (bill, index) { }}
+                            <li class="list-group-item support-bill-elem" id="support-bill-{{ index }}">
+                                <a href="{{ bill.url }}" target="_blank">{{ bill.title }}</a>
+                            </li>
+                            {{# }, this); }}
+                        </ul>
+                        {{# if (this.model.get('my_bills').supported_bills.length > 3) { }}
+                        <button type="button" class="support-button btn btn-light" onclick="toggleText('support')">Show more</button>
+                        {{# } }}
+                    </div>
+                    <div class="subtitle-lato">Bills I oppose</div>
+                    <div class="row">
+                        <ul class="list-group">
+                            {{# _.each(this.model.get('my_bills').opposed_bills, function (bill, index) { }}
+                            <li class="list-group-item oppose-bill-elem" id="oppose-bill-{{ index }}">
+                                <a href="{{ bill.url }}" target="_blank">{{ bill.title }}</a>
+                            </li>
+                            {{# }, this); }}
+                        </ul>
+                        {{# if (this.model.get('my_bills').opposed_bills.length > 3) { }}
+                        <button type="button" class="oppose-button btn btn-light" onclick="toggleText('oppose')">Show more</button>
+                        {{# } }}
                     </div>
                 </div>
+
             </div>
-            {{# } }}
+        </div>
     </div>
-</div>
 {% endverbatim %}
-{% endif %}
+
+
+<script>
+    var isSupportBillCollapsed = true;
+    var isOpposedBillCollapsed = true;
+
+    showAppropriateElements("oppose", isSupportBillCollapsed);
+    showAppropriateElements("support", isSupportBillCollapsed);
+
+    function showAppropriateElements(billType, isListCollapsed) {
+        var allElements = $('.' + billType +'-bill-elem');
+        var elementsToShow = allElements.filter(function() {
+            if(!isListCollapsed) {
+                return true;
+            }
+           return parseInt(this.id.replace(billType + '-bill-',''), 10) < 3;
```

Comment:
Add a comment or function docstring here describing the what this function does.